README.md

```diff
@@ -9,10 +9,16 @@ Setup
 
 ```
 cp settings.sh.example settings.sh
-# edit settings to update paths to foreman and proxy
+# edit settings to update paths to foreman and proxy 
+# check if port where smart proxy running  is same as port in settings.sh (e.g 8000)
```

Comment:
Can you update the ./settings.sh.example to use the 8000 port, which is the default port in the smart-proxy settings example?